nefertari/authentication/models.py

```diff
@@ -148,18 +148,22 @@ def get_authuser_by_name(cls, request):
             return cls.get_resource(username=username)
 
 
-def lower_strip(**kwargs):
-    return (kwargs['new_value'] or '').lower().strip()
+def lower_strip(event):
+    field = event.field
+    value = (field.new_value or '').lower().strip()
+    event.view._json_params[field.name] = value
```

Comment:
Added `set_field_value` method with params `value` and optional `field_name`. If `field_name` not provided, `self.field` is used if sef; if not set KeyError is raised.